### PR TITLE
Update CSV for 4.3.0

### PR DESCRIPTION
**Pull request by kshlm**
This PR updates the CSV to have a better description and remove stale examples.
It also changes the CSV version to 4.3.0 to follow the release version.

### CODE REVIEW

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -300,12 +284,81 @@ spec:
...
         version: v1
       version: v1alpha1
-  description: |2-
+  description: |2
+
+    Red Hat Openshift Container Storage (RHOCS) provides hyperconverged storage for applications within an Openshift cluster.
+
+    ## Components
+
+    RHOCS deploys 3 operators.
```
Comment (jarrpa) [inline deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: `s/3/three/`

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -300,12 +284,81 @@ spec:
...
+
+    ### OCS operator
+
+    The OCS operator is the primary operator for RHOCS. It serves to facilitate the other operators in OCS by performing administrative tasks outside their scope as well as watching and configuring their CustomResources.
+
+    ### Rook
+
+    Rook[1][1] deploys and manages Ceph on Openshift, which provides block and file storage.
```
Comment (jarrpa) [inline deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: Why are the links here twice?

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -300,12 +284,81 @@ spec:
...
+
+    ### NooBaa operator
+
+    The NooBaa operator deploys and manages NooBaa[2][2] on Openshift, which provides object storage.
```
Comment (jarrpa) [inline deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: Same thing here. Also, we should mention Multi-Cloud Gateway.

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -300,12 +284,81 @@ spec:
...
+    ```
+
+    Save the above as rhocs-namespace.yaml, and create the Namespace with,
+
+    ```
+    $ oc create -f rhocs-namespace.yaml
+    ```
+
+
+    ### OperatorGroup
```
Comment (jarrpa) [inline deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: Shouldn't the OLM handle the creation of the OperatorGroup?

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml [hunk 5]
```diff
@@ -300,12 +284,81 @@ spec:
...
+    Save the above as rhocs-operatorgroup.yaml, and create the OperatorGroup with,
+
+    ```
+    $ oc create -f rhocs-operatorgroup.yaml
+    ```
+
+    ## After subscription
 
-    Red Hat Openshift Container Storage (OCS) provides hyperconverged storage for applications within an Openshift cluster.
+    After the 3 operators have been deployed into the openshift-storage namespace, a StorageCluster can be created. Note that the StorageCluster resource is the only resource that a user should be creating. RHOCS includes many other custom resources which are internal and not meant for direct usage by users.
```
Comment (jarrpa) [inline deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: `s/3/three/`

File: deploy/olm-catalog/ocs-operator/ocs-operator.package.yaml [hunk 6]
```diff
@@ -1,4 +1,6 @@
 packageName: ocs-operator
 channels:
-- name: alpha
-  currentCSV: ocs-operator.v0.0.2
+- name: 4.2.0
+  currenCSV: ocs-operator.v0.0.1
```
Comment (jarrpa) [inline deploy/olm-catalog/ocs-operator/ocs-operator.package.yaml]: Should this be `v0.0.2`?

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml (hunk 2 above)
Comment (kshlm) [reply deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: I did it wrong. Should have been `[Rook][1]`.

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml (hunk 4 above)
Comment (kshlm) [reply deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: Nope. The OperatorGroup is created by the user. OLM installs into the operator group.

File: deploy/olm-catalog/ocs-operator/ocs-operator.package.yaml (hunk 6 above)
Comment (kshlm) [reply deploy/olm-catalog/ocs-operator/ocs-operator.package.yaml]: The 0.0.2 CSV no longer exists with this change. And release-4.2 has the CSV version set to 0.0.1.

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml (hunk 4 above)
Comment (umangachapagain) [reply deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: Is it the same with all the operators?

**Comment by kshlm** [comment]
@jarrpa Once this merges, could you also create a 4.3.0-alpha tag? Seems odd still having only a 0.0.1 tag.

**Comment by jarrpa** [comment]
Makes sense, sure. Do we need to edit something in openshift/release to get images pushed to that new tag?

**Comment by kshlm** [comment]
> Makes sense, sure. Do we need to edit something in openshift/release to get images pushed to that new tag?

Hopefully nothing. Images should already be tagged  v4.3.0. I need to check though. With the release-4.3 branch no longer following master, the images might not be updated.

**Pull request merged**